Updated ai-rag-injector docs (#8703)
Kong/docs.konghq.com: app/_hub/kong-inc/ai-rag-injector/how-to/_index.md

```

To omit the chunk content and only return the chunk ID, set `exclude_contents` to true.

> ## Update ingest content

Reviewer: ```suggestion
## Update content for ingesting
```

> You can update ingest content by sending request `/ai-rag-injector/:plugin_id/ingest_chunk` endpoint if you are runing Kong in traditional mode. However, this won't work for hybrid mode or Konnect because control plane can not access to the backend storage of the plugin. For this case, you can use the following script:

Reviewer: ```suggestion
If you are running {{site.base_gateway}} in traditional mode, you can update content for ingesting by sending a request to the `/ai-rag-injector/:plugin_id/ingest_chunk` endpoint. 

However, this won't work in hybrid mode or {{site.konnect_short_name}} because the control plane can't access the plugin's backend storage.

 To update content for ingesting in hybrid mode or {{site.konnect_short_name}}, you can use a script:
 
 1. Retrieve the ID of the AI RAG Injector plugin that you want to update.
 2. Copy and paste the following script to a local file, for example `ingest_update.lua`:
```

> ```

Reviewer: ````suggestion
```lua
````

  end
  metadata.embeddings_tokens_count = embeddings_tokens_count
  if #embeddings_vector ~= conf.vectordb.dimensions then
    return nil, "Embedding dimensions do not match the configured vector database. Embeddings were " ..
      #embeddings_vector .. " dimensions, but the vector database is configured for " ..
      conf.vectordb.dimensions .. " dimensions.", "Embedding dimensions do not match the configured vector database"
  end
  metadata.chunk_id = uuid()
  -- ingest chunk
  local _, err = vectordb_driver:insert(embeddings_vector, content, metadata.chunk_id)
  if err then
    return nil, "Failed to insert chunk: " .. err
  end
  ngx.update_time()
  metadata.ingest_duration = math.floor((ngx.now() - metadata.ingest_duration) * 1000)
  return metadata
end
assert(#args == 3, "2 arguments expected")
local plugin_id, content = args[2], args[3]
local plugin = get_plugin_by_id(plugin_id)
if not plugin then
  ngx.log(ngx.ERR, "Plugin not found")
  return
end
local metadata, err = ingest_chunk(plugin.config, content)
if err then
  ngx.log(ngx.ERR, "Failed to ingest: " .. err)
  return
end
ngx.log(ngx.INFO, "Update completed")

```

> Copy paste the whole content of the script to your local file, for example `ingest_update.lua`. You should retrieve the plugin id of the ai-rag-injector Plugin you are going to update before runing the run the script:

Reviewer: ```suggestion
```
The "before" portion should appear the script.

> ```
> kong runner ingest_api.lua <plugin_id> <content_to_update>
> ```

Reviewer: ````suggestion
3. Run the script from your Kong instance:
   
   ```sh
   kong runner ingest_api.lua <plugin_id> <content_to_update>
   ```
````
